api/swagger/swagger.yaml

```diff
@@ -588,6 +588,30 @@ paths:
           schema:
             $ref: "#/definitions/ErrorResponse"
 
+    /analytics:
+      x-swagger-router-controller: media
+      post:
+        operationId: uploadAnalytics
+        description: Uploads offline analytics
+        security:
+          - Bearer: []
+        x-security-scopes:
+          - admin
+          - user
+        consumes:
+          - multipart/form-data
```

Comment:
Is this correct for the data you want to upload ?

Reply:
Yes - patterned off media upload